include/extractor/guidance/roundabout_handler.hpp

```diff
@@ -37,18 +37,25 @@ class RoundaboutHandler : public IntersectionHandler
   public:
     RoundaboutHandler(const util::NodeBasedDynamicGraph &node_based_graph,
                       const std::vector<QueryNode> &node_info_list,
-                      const util::NameTable &name_table);
+                      const util::NameTable &name_table,
+                      const SuffixTable &street_name_suffix_table);
 
     ~RoundaboutHandler() override final;
 
     // check whether the handler can actually handle the intersection
-    bool canProcess(const NodeID from_nid, const EdgeID via_eid, const Intersection &intersection) const override final;
+    bool canProcess(const NodeID from_nid,
+                    const EdgeID via_eid,
+                    const Intersection &intersection) const override final;
 
     // process the intersection
-    Intersection operator()(const NodeID from_nid, const EdgeID via_eid, Intersection intersection) const override final;
+    Intersection operator()(const NodeID from_nid,
+                            const EdgeID via_eid,
+                            Intersection intersection) const override final;
 
   private:
-    detail::RoundaboutFlags getRoundaboutFlags(const NodeID from_nid, const EdgeID via_eid, const Intersection &intersection) const;
+    detail::RoundaboutFlags getRoundaboutFlags(const NodeID from_nid,
+                                               const EdgeID via_eid,
+                                               const Intersection &intersection) const;
```

Comment:
well, formatting changed with at least you switching to 3.8. I am on the version that you and I compared for equality, so ... your guess is as good as mine